### ISSUE TITLE
No esta funcionando el despliegue en Vercel

### ISSUE THREAD

**Issue by serudda** [serudda/reward-system#65]
### Descripción de la tarea

Por alguna razón, cuando cambiamos la estructura del proyecto, dejo de funcionar el despliegue a Vercel.
Es necesario corregir este error, para cuando queramos deployar el proyecto, y necesitemos probar: App y Bot.

La manera de probar este issue, es crearse un vercel de pruebas, con su user personal, y en el archivo .env guardar las variables de entorno requeridas:
``` shell
DATABASE_URL='<URL>'

NEXTAUTH_SECRET="<ANY_STRING>"
NEXTAUTH_URL="http://localhost:3000"

DISCORD_CLIENT_ID="<ID>"
DISCORD_CLIENT_SECRET="<SECRET_TOKEN>"
DISCORD_SERVER_ID="<SERVER_ID>"
DISCORD_BOT_TOKEN="<TOKEN>"
PREFIX="!"
```

No estoy seguro si tienes que crearte una base de pruebas en supabase, y un Bot de Discord de pruebas... Si tienes algún problema con esto, escribir a @serudda en Discord.

-----

### Criterios de aceptación

- [ ] Debe evidenciarse que el despliegue en el vercel de pruebas es exitoso. (se ve en los indicadores del PR, adj imagen):
![image](https://user-images.githubusercontent.com/10075532/224842150-9210b9d5-2055-4366-89d2-751cff4f44ea.png)

- [ ] Adjuntar el paso a paso de como configurar el Vercel.

**Comment by mastoreto** [comment]
Tomo el Issue

**Comment by yebt** [comment]
¿Hola, una pregunta, me podrías dar info adicional acerca del error que te arroja Vercel al tratar de desplegar?

**Comment by mastoreto** [comment]
Escribime en discord, y te paso todo el detalle

**Comment by mastoreto** [comment]
@serudda tenemos un bloqueante, Andry estuvo leyendo, y parece que al final Vercel no es amigable con los bots, ya que los toma como "servidores", por lo cual hay que evaluar otro método de alojamiento para el bot y configurar su despliegue.

Dejo la evidencia que Andry consiguió

[https://github.com/vercel/vercel/discussions/7767](url)

**Comment by victoruramayo** [comment]
@serudda Solo para tener un track otra posible solución al issue y en adición al detalle del bot con vercel, (no sé si el bot se logra correr en vercel) , logre hacer deploy de todo el monorepo en railway, tanto del nextjs como del bot

**Comment by serudda** [comment]
uh interesante @victoruramayo tienes el repo, o la configuración documentada en algún lugar?, así puedo ver cómo sería la configuración del despliegue del bot

**Comment by victoruramayo** [comment]
@serudda Claro, en un momento más hago un PR para que se pueda visualizar la información del despligue en el .README
